providers/openfeature-sdk-meta_provider/README.md

```diff
@@ -0,0 +1,54 @@
+# OpenFeature::SDK::Provider::MetaProvider for Ruby
+
+The `OpenFeature::SDK::Provider::MetaProvider` is a utility provider implementation that takes multiple [providers](https://docs.openfeature.dev/docs/specification/sections/providers) for use during flag resolution.
```

Comment:
Could you please add one more sentence explaining the use cases? It may not be obvious at first glace.